EIPS/eip-4844.md

```diff
@@ -50,15 +50,16 @@ Compared to full data sharding, this EIP has a reduced cap on the number of thes
 | `POINT_EVALUATION_PRECOMPILE_GAS` | `50000` |
 | `MAX_BLOBS_PER_BLOCK` | `16` |
 | `TARGET_BLOBS_PER_BLOCK` | `8` |
-| `MAX_BLOBS_PER_TX` | `2` |
-| `GASPRICE_UPDATE_FRACTION_PER_BLOB` | `64` |
+| `MIN_DATA_GASPRICE` | `10**13` |
+| `DATA_GASPRICE_UPDATE_FRACTION` | `84` |
 | `MAX_VERSIONED_HASHES_LIST_SIZE` | `2**24` |
 | `MAX_CALLDATA_SIZE` | `2**24` |
 | `MAX_ACCESS_LIST_SIZE` | `2**24` |
 | `MAX_ACCESS_LIST_STORAGE_KEYS` | `2**24` |
 | `MAX_TX_WRAP_KZG_COMMITMENTS` | `2**24` |
 | `LIMIT_BLOBS_PER_TX` | `2**24` |
-| `GAS_PER_BLOB` | `120000` |
+| `DATA_GAS_PER_BLOB` | `1` |
```

Comment:
Are we okay with defining as 1 and closing the door on forward compatibility of cheaper data-gas items? It enshrines blobs the smallest theoretical unit. That is, you can't price something as costing less than a blob without repricing blobs. 

I don't know what that smaller data-gas item might be so I lean toward this being okay and nice for easy accounting but I could have  a blind spot

Reply:
Agreed, seems sensible to move to a "1 gas per byte" model, i.e. `DATA_GAS_PER_BLOB = 2 ** 17`. That also makes a potential move to time-based targeting easier.